mypy.ini

```diff
@@ -5,6 +5,7 @@
 plugins = mypy_plugins/check_mypy_version.py, numpy.typing.mypy_plugin
 
```

Comment:
mypy-strict and mypy-nofollow are two separate inis. Don't you need to edit those too?

Reply:
I wasn't aware of those files. I'm sure we would want to be consistent across them so I'll add it.